tests/yzalis/metadata.json

```diff
@@ -0,0 +1,4 @@
+{
+    "name": "ZSXSoft",
```

Comment:
This needs to be changed to match the test suite provider :)

Reply:
fixed